webapp/client/src/components/FormFieldYesNo.test.js

```diff
@@ -96,18 +96,21 @@ describe("FormFieldYesNo", () => {
 
     it("Should focus Nein Field when pressing tab and then right arrow key", () => {
       userEvent.tab();
+      // This is currently the only way to trigger an arrow event. userEvent.keyboard('{ArrowRight}'); is not supported yet.
```

Comment:
Can we please not spam these comments everywhere? No one will ever remove them again and I don't think they help us…

Reply:
Mh, probably not, no. :/